### ISSUE TITLE
Isn't possible start after update from 2.0.23 -> 2.0.25

### ISSUE THREAD

**Issue by vitorjuanh** [open-wa/wa-automate-nodejs#866]
**Describe the bug**
After update to 2.0.25 isn't possible to start the script.

**Steps to Reproduce**
1. Usign the first example from open-wa/wa-automate-nodejs at index.js file
2. Run lib install:  npm i --save @open-wa/wa-automate
2. Start the app: node index.js
```
(node:10432) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'sessionId' of undefined
    at Object.<anonymous> (/home/user/temp/node_modules/@open-wa/wa-automate/dist/controllers/initializer.js:117:44)
    at step (/home/user/temp/node_modules/@open-wa/wa-automate/dist/controllers/initializer.js:63:23)
    at Object.next (/home/user/temp/node_modules/@open-wa/wa-automate/dist/controllers/initializer.js:44:53)
    at fulfilled (/home/user/temp/node_modules/@open-wa/wa-automate/dist/controllers/initializer.js:35:58)
```

**create() code**
This is the code you use to create the client. e.g
```javascript
wa.create().then(client => start(client));

function start(client) {
  client.onMessage(message => {
    if (message.body === 'Hi') {
      client.sendText(message.from, '👋 Hello!');
    }
  });
}
```

**Expected behavior**
Apear QR Code.

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
Debug Info {
  WA_VERSION: 2.0.25,
  PAGE_UA: ...,
  WA_AUTOMATE_VERSION: ...,
  BROWSER_VERSION: ...
}
```

**Screenshots**
![image](https://user-images.githubusercontent.com/57458936/94345429-b2caea80-fff3-11ea-97db-d4ad79b00f1b.png)

**Host (please complete the following information):**

**Additional context**

**Comment by OliverNoMore** [comment]
It's happening to me on clean install, already tried reinstalling another virtual machine and still. Running Xubuntu and did the same proccedure

**Comment by smashah** [comment]
This will be fixed in the next version. For now do this:

```javascript
wa.create({})
```

**Comment by contatodct** [comment]
could you show me a complete example of how it would work here? can you make corrections for phoning?

const wa = require('@open-wa/wa-automate');

wa.create().then(client => start(client));

function start(client) {
  client.onMessage(message => {
    if (message.body === 'Hi') {
      client.sendText(message.from, '👋 Hello!');
    }
  });
}

**Comment by smashah** [comment]
@contatodct fixed your code

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.0.26](https://github.com/open-wa/wa-automate-nodejs/compare/2.0.25...2.0.26) (2020-09-28)

- fixes #866 [`#866`](https://github.com/open-wa/wa-automate-nodejs/issues/866)
- Release 2.0.26 [`fb3607d`](https://github.com/open-wa/wa-automate-nodejs/commit/fb3607da628be19b36cc9100f6d1f3884af4de04)
